### PR TITLE
4.5:add winch options

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-daiwa-winch.rst [hunk 1]
```diff
@@ -66,8 +66,23 @@ Set the following parameters
...
 
+Winch Options
+=============
+
+The :ref:`WINCH_OPTIONS<WINCH_OPTIONS>` parameter provides a bit mask of possible configuration options for the winch when the following bits are set in the parameter:
+
+===     ======
+Bit     Option
+===     ======
+0       Spin freely on startup allows the line to be pulled out after startup. This is the current default but is actually quite annoying because we need to remember to disengage the clutch before takeoff or the package will likely be left on the ground.
```
Comment (rmackay9) [inline common/source/docs/common-daiwa-winch.rst:77]: We can shorten this one to just, "Spin freely on startup allows the line to be pulled out after startup"

Comment (Hwurzburg) [reply common/source/docs/common-daiwa-winch.rst:77]: will do

**Comment by Hwurzburg** [comment]
its a 4.5 merge....and is already linked in upcoming...all marked 4.5 wiki are there

**Comment by rmackay9** [comment]
ok, so I would have expected that I could find the new lidar wiki page in the upcoming section. https://ardupilot.org/copter/docs/common-master-features.html... but I guess you're thinking we should have a page in the wiki at all until we've started 4.5 beta testing?

**Comment by Hwurzburg** [comment]
as I said its already been linked
Capture

**Pull request merged**